zb-db/src/main/java/io/zeebe/db/impl/rocksdb/transaction/ZeebeTransactionDb.java

```diff
@@ -136,6 +143,17 @@ protected ZeebeTransactionDb(
     this.handelToEnumMap = handelToEnumMap;
     this.closables = closables;
     this.columnFamilyNamesClass = columnFamilyNamesClass;
+
+    prefixReadOptions = new ReadOptions().setPrefixSameAsStart(true).setTotalOrderSeek(false);
+    closables.add(prefixReadOptions);
+    simpleReadOptions = new ReadOptions();
+    closables.add(simpleReadOptions);
+    final WriteOptions simpleWriteOptions = new WriteOptions();
+    closables.add(simpleWriteOptions);
+
+    final Transaction transaction = optimisticTransactionDB.beginTransaction(simpleWriteOptions);
+    currentTransaction = new ZeebeTransaction(transaction);
+    closables.add(currentTransaction);
```

Comment:
This closables are closed in order they were added to the list as far as I can tell, which means that the options are closed before the transaction and database and so on. Can this lead to problems? I would have expected a reversed order when closing. If on closing an exception happens we stop closing the others and just throw a `RuntimeException`, shouldn't we try to close all of them?

Reply:
I think you're right we did the reverse closing before the migration to the zb-db module, maybe I missed that. Will add this again.Yeah might make sense.